plugins/explore/src/components/ToolExplorerContent/ToolExplorerContent.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  Content,
+  ContentHeader,
+  EmptyState,
+  Progress,
+  SupportButton,
+  useApi,
+} from '@backstage/core';
+import React from 'react';
+import { useAsync } from 'react-use';
+import { exploreToolsConfigRef } from '../../../../explore-react/src';
```

Comment:
oops, should be a module import

Reply:
😱  Hmm, why doesn't any tooling detected this... 😆

Reply:
VSCode loves to do it when moving stuff around...

Reply:
Yeah I know. I thought we had a lint check for this, actually. I must be misremembering

Reply:
No we do!

```ts
    // Avoid cross-package imports
    'no-restricted-imports': [2, { patterns: ['**/../../**/*/src/**'] }],
```

wonder why that didn't fire ...

Reply:
Maybe because the `/**` at then end?

Reply:
yeah that sounds sensible